---
content_type: page
title: Lecture Summaries
uid: 5f3c6b9c-7372-fc45-8a15-04c08d223039
---

| WEEK # | TOPICS | LECTURE SUMMARIES{{< anchor "1" >}}{{< /anchor >}} |
| --- | --- | --- |
| 1 | Introduction to the course | First, we will get to know each other! Then we will go over the syllabus, including expectations and goals of the class, an overview of the course, and an introduction to the first two papers. We will also discuss strategies for reading primary scientific papers.{{< anchor "2" >}}{{< /anchor >}} |
| 2 | The discovery of RNA interference | Introduction to RNAi. These two papers describe for the first time the phenomenon of RNA interference and open up the field of RNAi. Interestingly, the two studies use very different model systems, highlighting how in science there is more than one way to make a groundbreaking discovery!{{< anchor "3" >}}{{< /anchor >}} |
| 3 | Endogenous miRNA | Before the discovery of RNAi by Fire and Mello, a microRNA had previously been identified but its function and mechanism of action were not understood. The first paper describes the cloning of a small, noncoding gene (the first miRNA!) from a genetic screen. The second paper reports the second example of an endogenous miRNA. This paper helped to establish the evolutionarily conserved and broad importance of miRNAs.{{< anchor "4" >}}{{< /anchor >}} |
| 4 | miRNA biogenesis pathway | Small RNAs require several proteins to carry out their important functions. We will discuss a few of the key catalytic enzymes in the RNAi pathway. The first paper describes Dicer, an enzyme that is central to producing small RNAs that mediate RNAi. The second paper describes Argonaute, a protein that assists targeting of mRNAs by siRNAs and miRNAs for post-transcriptional gene regulation.{{< anchor "5" >}}{{< /anchor >}} |
| 5 | Harnessing RNAi as a tool | In addition to playing important biological functions, the RNAi pathway can be exploited in the lab for experimental analysis of gene function. We will discuss the two main methods of gene knockdown by RNAi. The first paper investigates the use of synthetic siRNAs, generated through _in vitro_ chemical synthesis. The second paper describes the use of expression constructs (e.g. DNA plasmids) that encode siRNA precursors and are expressed _in vivo._{{< anchor "6" >}}{{< /anchor >}} |
| 6 | miRNAs in development | Functional studies of newly identified miRNAs have revealed their importance in defining cell fates in embryonic and adult tissues. We will discuss the role of miRNAs in two different paradigms in which loss-of-function of miRNAs led to disruption in cardiac development and B cell differentiation.{{< anchor "7" >}}{{< /anchor >}} |
| 7 | miRNAs in cancer: tumor suppressors and oncogenes | miRNA function is dysregulated in various diseases, including cancer. We will discuss the role of miRNAs in cancer with examples of tumor suppressive and oncogenic miRNAs. The first paper is about a cluster of tumorigenic miRNAs that induce B cell lymphoma. The second paper is about _let-7_, one of the most well–studied tumor suppressor genes, and its role in suppressing lung tumor development.{{< anchor "8" >}}{{< /anchor >}} |
| 8 | RNAi therapeutics | From the study of siRNAs and miRNAs in various biological settings, it became increasingly clear that they could be utilized as therapeutic tools against various diseases. miRNAs and siRNAs are very similar in their biogenesis; however their target specificities are different, making them two distinct therapeutic tools. In this class we discuss a study that uses siRNAs against apoB and another using liver-specific miR-122 to reduce cholesterol levels in two different models of hypercholesterolaemia.{{< anchor "9" >}}{{< /anchor >}} |
| 9 | Visit to Alnylam | A first-hand look at a player in the newly-emerging private sector pursuing RNAi therapeutics.{{< anchor "10" >}}{{< /anchor >}} |
| 10 | Other classes of small noncoding RNAs: piRNAs | [PIWI-interacting RNAs](http://en.wikipedia.org/wiki/Piwi-interacting_RNA) (piRNAs) are a distinct class of small non-coding RNAs in the germ line of many animal species. First, we will discuss a study that identifies mammalian piRNAs and their machinery. This study also describes their role in maintaining genome-stability in the germ line by epigenetic silencing of transposable elements. To understand the origin and evolution of piRNAs, we will examine a recent study that determines how piRNAs respond to the presence of new transposable elements in the germ line.{{< anchor "11" >}}{{< /anchor >}} |
| 11 | Long noncoding RNAs: XIST | The phenomenon of X chromosome inactivation was discovered almost half a century ago, but it has taken several decades to discover that this process is governed largely by long noncoding RNAs. In particular, studies of the noncoding RNA XIST, which is required for X chromosome inactivation, have served as a foundation for the field of noncoding RNAs. The first paper describes some of the properties of the XIST gene, whereas the second paper addresses the mechanism of how XIST inactivates the X chromosome.{{< anchor "12" >}}{{< /anchor >}} |
| 12 | Long noncoding RNAs: lincRNAs | In recent years, technological advances in RNA sequencing technologies have revealed that a large portion of the genome is transcribed into noncoding RNAs of various lengths. The first paper describes the recently discovered class of large intervening noncoding RNAs, or lincRNAs. In this study, the authors identify more than 5000 lincRNAs, the vast majority of which have no known function. The second paper assigns a function to one lincRNA, HOTAIR, in regulating gene expression during development.{{< anchor "13" >}}{{< /anchor >}} |
| 13 | Final class | After the oral presentations, we will discuss an overview of what has been learned about noncoding RNAs.